pkg/daemon/controller/controller_test.go

```diff
@@ -648,6 +648,14 @@ func TestReconcile(t *testing.T) {
 							metadata.DesiredVersionAnnotation: desiredVersion,
 						},
 					},
+					Status: core.NodeStatus{
```

Comment:
Can you explain why this only causes a 1 min wait and not a failure?

Reply:
Ah looks like we are missing an assert.NoError() below